README.md

```diff
@@ -10,21 +10,29 @@ You can reset the Vega Editor by going to https://vega.github.io/editor/#/reset
 
 ## Usage Instructions
 
-To run the editor locally, you must first install the dependencies and then launch a local web server. We assume you have [yarn](https://yarnpkg.com/) installed.
+### In Short
```

Comment:
Maybe rename this to "running with docker"

Reply:
Updated to "_Run Vega-Editor With Docker_"